conda_package/mpas_tools/mesh/creation/sort_mesh.py

```diff
@@ -0,0 +1,187 @@
+
+import numpy as np
+import os
+import xarray
+import argparse
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import reverse_cuthill_mckee
+
+
+def sort_fwd(data, fwd):
+    vals = data.values
+    vals = vals[fwd - 1]
+    return vals
+
+
+def sort_rev(data, rev):
+    vals = data.values
+    mask = vals > 0
+    vals[mask] = rev[vals[mask] - 1]
+    return vals
+
+
+def sort_mesh(mesh):
+    """
+    SORT-MESH: sort cells, edges and duals in the mesh
+    to improve cache-locality.
+
+    """
```

Comment:
Here's the docstring format we use in MPAS-Tools:
```suggestion
    """
    Sort cells, edges and duals in the mesh
    to improve cache-locality.

    Parameters
    ----------
    mesh : xarray.Dataset
        A dataset containing an MPAS mesh to sort
        
    Returns
    -------
    mesh : xarray.Dataset
        A dataset containing the sorted MPAS mesh
    """
```